packages/ui-react/src/components/LanguageMenu/LanguageMenu.tsx

```diff
@@ -1,8 +1,13 @@
-import React from 'react';
-import classNames from 'classnames';
 import type { LanguageDefinition } from '@jwp/ott-common/types/i18n';
+import classNames from 'classnames';
+import { t } from 'i18next';
```

Comment:
We use the `useTranslation` hook to get a translation value
```suggestion
import { useTranslation } from 'react-i18next';
```

Further down your code you can use the following:
```
const { t } = useTranslation('menu');
```

To get the translation from the `menu.json` file